src/vmm/src/vcpu/mod.rs

```diff
@@ -57,6 +57,7 @@ pub type Result<T> = result::Result<T, Error>;
 /// This struct is a temporary (and quite terrible) placeholder until the
 /// [`vmm-vcpu`](https://github.com/rust-vmm/vmm-vcpu) crate is stabilized.
 pub(crate) struct Vcpu {
+    id: u8,
```

Comment:
Should we make this `usize` and save ourselves future extensions for >255 vCPUs?

Reply:
Right now we cannot make this an u32 because the kvm-ioctls crate expects an u8 here. We first need to update the kvm-ioctls crate, and then we can also increase this number. Btw, KVM API defines this as an u64, so I think we should just do the same. I've opened an issue for this (https://github.com/rust-vmm/kvm-ioctls/issues/123), but I wouldn't block on the kvm-ioctls version to be published if you're also okay with that.

Reply:
That's fine with me.